components/detournavigator/debug.hpp

```diff
@@ -20,7 +23,11 @@ namespace DetourNavigator
     class Log
     {
     public:
-        Log() : mEnabled(false) {}
+        Log()
+            : mEnabled(false)
```

Comment:
Looks like VS 2013 can not initilize the std::atomic_bool variable by bool "false" value.